graviti/schema/catalog_to_schema.py

```diff
@@ -0,0 +1,324 @@
+    def __call__(  # type: ignore[override]
+        self, subcatalog: InstanceMaskSubcatalog, labels: List[_LabelBase]
+    ) -> Dict[str, Any]:
+        field = super().__call__(subcatalog, labels)
+        is_tracking = subcatalog.is_tracking
+        if is_tracking:
+            field["isTracking"] = is_tracking
+        return field
+
+
+class _ConverterPanopticMask(_ConverterSingleLabel):  # pylint: disable=too-few-public-methods
+    def __call__(  # type: ignore[override]
+        self, subcatalog: PanopticMaskSubcatalog, labels: List[_LabelBase]
+    ) -> Dict[str, Any]:
+        field = super().__call__(subcatalog, labels)
+        field["stuffIds"] = field.pop("categoryIds")
+        return field
+
+
+class _ConverterMultiLabel(_ConverterCommon):  # pylint: disable=too-few-public-methods
+    def __init__(self, label_type: str, portex_type: str) -> None:
+        super().__init__(label_type, portex_type)
+        self._field_name = label_type.upper() if label_type == "rle" else f"{label_type}s"
+
+    def __call__(  # type: ignore[override]
+        self, subcatalog: SubcatalogBase, labels: List[_LabelBase]
+    ) -> Dict[str, Any]:
+        items = {"type": self._portex_type, **_get_category_and_attribute_params(subcatalog)}
+        return {"name": self._field_name, "type": "array", "items": items}
+
+
+class _ConverterMultiGeometryLabel(_ConverterMultiLabel):  # pylint: disable=too-few-public-methods
+    def __init__(self, label_type: str, portex_type: str, label_value_level: int) -> None:
+        super().__init__(label_type, portex_type)
+        self._label_value_level = label_value_level
+
+    def __call__(  # type: ignore[override]
+        self, subcatalog: SubcatalogBase, labels: List[_LabelBase]
+    ) -> Dict[str, Any]:
+        field = super().__call__(subcatalog, labels)
+
+        if labels:
+            value = labels
+            for _ in range(self._label_value_level):
+                value = value[0]  # type: ignore[assignment]
+            field["items"].update(_get_dtype(value))  # type: ignore[arg-type]
+
+        return field
+
+
+class _ConverterMultiKeypoints2D(
+    _ConverterMultiGeometryLabel
+):  # pylint: disable=too-few-public-methods
+    def __call__(  # type: ignore[override]
+        self, subcatalog: Keypoints2DSubcatalog, labels: List[LabeledKeypoints2D]
+    ) -> Dict[str, Any]:
+        field = super().__call__(subcatalog, labels)  # type: ignore[arg-type]
+
+        items = field["items"]
+        keypoints = subcatalog.keypoints[0]
+        items["number"] = keypoints.number
+        if hasattr(keypoints, "names"):
+            items["names"] = keypoints.names
+        if hasattr(keypoints, "skeleton"):
+            items["skeleton"] = keypoints.skeleton
+        if hasattr(keypoints, "visible"):
+            items["visible"] = keypoints.visible
+
+        return field
+
+
+_ConverterClassification("classification")
```

Comment:
it is not good to register the instance in the `__init__`
will it be better to register them here:
```python
_LABEL_PROCESSORS["classification"] = _ConverterClassification()
```

Reply:
In this way, the label type will be written twice.
`_LABEL_PROCESSORS["classification"] = _ConverterClassification("classification")`

Reply:
we we really need to store label type in the instance?

Reply:
The label type is for the field name.